Modules/resource.c

```diff
@@ -487,8 +472,27 @@ PyInit_resource(void)
         v = PyLong_FromLong((long) RLIM_INFINITY);
     }
     if (v) {
-        PyModule_AddObject(m, "RLIM_INFINITY", v);
+        PyModule_AddObject(module, "RLIM_INFINITY", v);
```

Comment:
What happens if this call fails? Py_DECREF(v) is needed on error, no?

Reply:
make sense, it will raise refleaks when it failed.